omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -188,7 +188,44 @@
                 SupportsInformationals="yes"
                 SupportsWarnings="yes"/>
         </Component>
+      </Directory>
+      <Directory Id="BIN" Name="bin">
+        <Directory Id="AGENT" Name="agent">
+          <!-- Windows service declaration for APM agent -->
+          <Component Id="DATADOGAPMSERVICE" Guid="a8e611fa-aedb-4c16-969c-bb827af0bd53">
+            <File Id="trace_agent.exe"
+                Name="trace-agent.exe"
+                Vital="yes"
+                KeyPath="yes"
+                DiskId="1"
+                Source="$(var.InstallDir)/bin/agent/trace-agent.exe"/>
+            <ServiceInstall Id="DatadogAPMServiceInstaller"
+                            DisplayName="Datadog Trace Agent"
+                            Description="Send tracing metrics to Datadog"
+                            Name="datadog-trace-agent"
+                            ErrorControl="ignore"
+                            Start="demand"
+                            Type="ownProcess"
+                            Vital="yes"
+                            Interactive="no"
+                            Account="LocalSystem"
+                            Arguments="-ddconfig=c:\programdata\datadog\trace-agent.conf" >
+                <ServiceDependency Id="datadogagent" />
```

Comment:
does this have the same limitation as the current agent 5 apm service, i.e. if the main agent service is stopped from the Task Manager (instead of the SCM) while the apm service is running, the user gets an error stating that the service can't be stopped because a dependent service is still running?

Reply:
Yes.  From the command line (or the SCM gui) then it will say "if you stop datadog agent, then datadog log agent will also be stopped"...

When we stop services ourselves, we'll probably need to add on the stopping of dependent services.